reference/configuration: update the description of the default value of unify-read-pool (#2063)
pingcap/docs: reference/configuration/tikv-server/configuration-file.md

### `unify-read-pool`

+ Whether to use a single thread pool to serve all the read requests.

> + Default value: `true` if there are no customized configurations under the `readpool.storage` and `readpool.coprocessor` sections, otherwise `false`.

Reviewer: ```suggestion
+ Default value: `true` when there are no customized configurations in the `readpool.storage` and `readpool.coprocessor` sections. Otherwise, the value is `false`.
```

Author: Changed. Thanks.
